network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -483,6 +516,15 @@ class Homepage(FoundationMetadataPageMixin, Page):
           heading='hero',
           classname='collapsible'
         ),
+        MultiFieldPanel(
+          [
+            ImageChooserPanel('spotlight_image'),
+            FieldPanel('spotlight_headline'),
+            InlinePanel('spotlight_posts', label='Posts', min_num=3, max_num=4),
+          ],
+          heading='spotlight',
+          classname='collapsible'
```

Comment:
I don't recall if this was the one we wanted to have collapsed by default? If it is, you can append `collapsed` as a `classname`. If it's not, ignore this.

Reply:
these didn't look quite long enough, but let's put that question to the design team once we have the homepage backend changes all landed in, and they can see what feels right to them.